ci-operator/step-registry/upi/install/azure/upi-install-azure-commands.sh

```diff
@@ -112,6 +112,9 @@ AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
 if [[ -f "${SHARED_DIR}/azure_minimal_permission" ]]; then
   echo "Setting AZURE credential with minimal permissions to install UPI"
   AZURE_AUTH_LOCATION="${SHARED_DIR}/azure_minimal_permission"
+elif [[ -f "${SHARED_DIR}/azure-sp-contributor.json" ]]; then
+  echo "Setting AZURE credential with Contributor role only to install UPI"
+  export AZURE_AUTH_LOCATION=${SHARED_DIR}/azure-sp-contributor.json
```

Comment:
so this means in 4.19+, the Azure UPI install will be either minimal permission install or Contributor role only install, right?

Reply:
yes, since user-assigned identity is also removed from arm template, related permissions are not required. 
BTW, identity creation can also be deleted in Azure UPI installation doc. I will add comment in epic to doc team for reference.